### ISSUE TITLE
npm test doesn't work with node 7+

### ISSUE THREAD

**Issue by michalbcz** [realreality/reality-plugin#62]
if you use node 7+ npm/yarn test doesn't work properly - Popup.spec.js tests fail

**Comment by michalbcz** [comment]
@vire what do you think? where is the problem?

**Comment by vire** [comment]
do you have a log/screenshot pls?

**Comment by vire** [comment]
Might be related to https://github.com/vuejs/vue/issues/4304

**Comment by vire** [comment]
thes work with 

`"node": "v7.2.0"`

```
"vue": "2.1.4",
"jest-vue-preprocessor": "^0.1.1",
"vue-template-compiler": "2.1.4",
```

**Comment by vire** [comment]
Fixed in #63

**Comment by michalbcz** [comment]
@vire nice work, thanks :)
